atm/Zolotarev/src/main/java/org/anzo/atm/Card.java

```diff
@@ -0,0 +1,54 @@
+package org.anzo.atm;
+
+import java.util.Currency;
+
+/**
```

Comment:
you don't need to put such comments into your files, the user information goes with the git commit